scripts/transition-github-release-note-state.sh

```diff
@@ -52,14 +52,14 @@ if [[ ${ASSERT_NO_RELEASE_NOTES_EXIST} == "true" ]]; then
   NUM_EXIST_RELEASE_NOTES=$(curl "${CURL_ARGS[@]}" | jq --arg tag "${TAG}" '[.[] | select ( .tag_name == $tag )] | length')
   if [[ ${NUM_EXIST_RELEASE_NOTES} -gt 0 ]]; then
       >&2 echo "${NUM_EXIST_RELEASE_NOTES} release note(s) already exist for tag ${TAG}"
-   exit -1
+   exit 11
```

Comment:
why 11?

Reply:
Completely arbitrary on my part, it looked the most similar to `-1`. I don't believe there are any standards other than `0` being success.

Reply:
The [Advanced Bash-Scripting Guide](https://tldp.org/LDP/abs/html/exitcodes.html) confirms my suspicions. We could use `1` as a generic error but its meaningless for debugging. I'm sticking to 11 so I can just merge this and end the pain.